packages/core/src/tools/ToolsManager.ts

```diff
@@ -0,0 +1,27 @@
+import type { BlockToolConstructor } from '../entities/BlockTool.js';
+import { Paragraph } from './internal/paragraph/index.js';
+import type { EditorConfig } from '@editorjs/editorjs';
+
+/**
+ * Works with tools
+ */
+export default class ToolsManager {
```

Comment:
Is this something we intend to keep? We could use ToolsFactory/ToolsCollection from the 2nd version